### ISSUE TITLE
loss value nan

### ISSUE THREAD

**Issue by kunsungwoo** [THU-MIG/yolov10#231]
![4](https://github.com/THU-MIG/yolov10/assets/137399781/2b57420f-5e2d-4cfe-a02b-78ed7e3717fa)

During training, the loss value changes to nan and learning is not possible. I remember this phenomenon also occurred when fine-tuning yolov7.

**Comment by Andy718811** [comment]
I also encounter this problem when using AdamW as the optimizer.
